Wrappers/Python/ccpi/viewer/CILViewer.py

```diff
@@ -491,12 +505,12 @@ def __init__(self, dimx=600,dimy=600, renWin=None, iren=None, ren=None, debug=Fa
 
         # Help text
         self.helpActor = vtk.vtkActor2D()
-        self.helpActor.GetPositionCoordinate().SetCoordinateSystemToNormalizedDisplay()
+        self.helpActor.GetPositionCoordinate(
+        ).SetCoordinateSystemToNormalizedDisplay()
```

Comment:
I really don't like this!!!

Reply:
I don't know why it didn't put an indent, that would be better